build/tools/mingw/parameters.bat

```diff
@@ -14,10 +14,13 @@ if "%2" == "" goto :NOPARAMS
 
 rem Define compiler's bin directory
 if "%1" == "1220" (
-    set COMPBINDIR=G:\msys64\mingw%2\bin
+    set COMPBINDIR=G:\msys64-gcc1220\mingw%2\bin
 ) else (
     set COMPBINDIR=G:\MinGW%1-%2\bin
 )
+if "%1" == "1320" (
```

Comment:
FWIW I think you can use `else if` in BAT files, so this could be done above instead of overwriting the value set in the line 19.